experimental/devices/nrzled/doc.go

```diff
@@ -8,6 +8,15 @@
 //
 // Note that some ICs are 7 bits with the least significant bit ignored, others
 // are using a real 8 bits PWM. The PWM frequency varies across ICs.
+// User can select a driver implementation using a memory-mapped GPIO line
+// or leverage a SPI connection's MISO pin.
+//
+// The SPI implementation is sensitive to variations in SPI clock speed.
+// On the Raspberry Pi, you will need to add `core_freq=250`
```

Comment:
That's the kind of information that would eventually should be put on the web site; https://periph.io/device/ with a new nrzled entry that. For now it's fine.

250 is already the default on RPi1 and 2. So you mean for RPi3, Zero and later? Can you expand on the glitching issue?

Reply:
I did my testing on a Pi0/0W, and saw random "sparkles" of white on my strip around 20% of the time. 
Turns out that, while  Neopixels are more [timing-tolerant](https://wp.josh.com/2014/05/13/ws2812-neopixels-are-not-so-finicky-once-you-get-to-know-them/) than most people expect, variations in the core freq threw off the timing enough to flip bits in the transmission. [This discussion](https://github.com/raspberrypi/linux/issues/2060) gave me the clue to lock core_freq

Reply:
Thanks, this can be put it here for now, and moved later.